### DISCUSSION TITLE
begging for keycloak 7.0.0 document

### DISCUSSION THREAD

**Discussion by sabbatical66** [keycloak/keycloak#33324] [Q&A] [answered]
I have a cluster service using Keycloak 7.0.0 for authentication. I want to develop SMS verification functionality, but I cannot find the documentation for Keycloak 7.0.0. Could an expert provide a web link or a PDF? If it's not convenient to post it here, could an expert send it to the email address [1779475106@qq.com]? Thank you very so much~

**Comment by jonkoops** [comment, marked as answer]
Keycloak 7 has been end-of-life for a very long time and is vulnerable to several critical security issues. You should not be running it in any production environment. @dasniko has [an excellent tutorial](https://www.n-k.de/2020/12/keycloak-2fa-sms-authentication.html) on this subject.

**Reply by codespearhead** [reply]
This question has been asked before. Do you mind marking those two comments as the answer to their respective discussion if you agree with them?

[1] https://github.com/keycloak/keycloak/discussions/33324#discussioncomment-10824977
[2] https://github.com/keycloak/keycloak/discussions/26706#discussioncomment-9001438

**Comment by sabbatical66** [comment]
Thank you very much for your reply！
I am also in a difficult situation. I have taken over a very complex project where Keycloak serves as the authentication center for several services. The project has developed several plugins for Keycloak 7.0. I attempted to upgrade, but there were too many errors, so I had to give up. Personally, I cannot handle the workload of upgrading Keycloak. 
If there are any documentation leads, I would appreciate it if you could inform me. Thank you again for your reply, and I am very grateful.




------------------&nbsp;原始邮件&nbsp;------------------
发件人:                                                                                                                        "keycloak/keycloak"                                                                                    ***@***.***&gt;;
发送时间:&nbsp;2024年9月27日(星期五) 晚上6:56
***@***.***&gt;;
***@***.******@***.***&gt;;
主题:&nbsp;Re: [keycloak/keycloak] begging for keycloak 7.0.0 document (Discussion #33324)





 
Keycloak 7 has been end-of-life for a very long time and is vulnerable to several critical security issues. You should not be running it in any production environment. @dasniko has an excellent tutorial on this subject.

**Reply by codespearhead** [reply]
As [jonkoops](https://github.com/jonkoops) noted, You should upgrade to the latest stable version as soon as it becomes available, as only the latest release will receive updates [1][2].

As for the question itself, it's a duplicate of [3][4].

[1] https://github.com/keycloak/keycloak/discussions/29126
[2] https://github.com/keycloak/keycloak/discussions/29126#discussioncomment-9257666
[3] https://github.com/keycloak/keycloak/discussions/26706
[4] https://github.com/keycloak/keycloak/discussions/26706#discussioncomment-9001438

**Reply by sabbatical66** [reply]
I am very sorry for the late reply. I was unable to log in to GitHub for a while due to network issues. Now that I have logged in, I can't find my issue (I feel so silly). Thank you very much for your help. I really appreciate it.

**Comment by sabbatical66** [comment]
Yes please.&nbsp;I am very sorry that I couldn't find this issue. I have already resolved it, and I am very grateful. I am still confused about why I couldn't find my own issue, and I replied via email.




------------------&nbsp;原始邮件&nbsp;------------------
发件人:                                                                                                                        "keycloak/keycloak"                                                                                    ***@***.***&gt;;
发送时间:&nbsp;2024年10月3日(星期四) 凌晨4:14
***@***.***&gt;;
***@***.******@***.***&gt;;
主题:&nbsp;Re: [keycloak/keycloak] begging for keycloak 7.0.0 document (Discussion #33324)





 
This question has been asked before. Do you mind marking those two comments as the answer to their respective discussions if you agree with them?
 
[1] #33324 (reply in thread)
 [2] #26706 (comment)
